### ISSUE TITLE
Failed to install `prestissimo` plugin since version 2.20.0

### ISSUE THREAD

**Issue by s4nji** [shivammathur/setup-php#612]
**Describe the bug**
`prestissimo` plugin could not be installed since the recent `2.20.0` action update.  
It can still be installed successfully in `2.19.1`.

**Version**
- [x] `v2`
- [x] `2.20.0`
- [ ] `2.19.1`

**Runners**

- [x] GitHub Hosted
- [x] Self Hosted

**Operating systems**
- [x] Ubuntu 22.04
- [x] Ubuntu 20.04

**PHP versions**
- [x] 7.0

**To Reproduce**
```yml
      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '7.0'
          tools: 'composer:v1, prestissimo'
          coverage: 'none'
```
This fails; you can see an example of the failing workflow run with github hosted runners [here](https://github.com/test-org-abc123/test-repo-one/runs/7212154899?check_suite_focus=true#step:2:15) (workflow definition [here](https://github.com/test-org-abc123/test-repo-one/actions/runs/2621884242/workflow))

**Expected behavior**
Plugin installed successfully as with `2.19.1`:  
Example workflow run [here](https://github.com/test-org-abc123/test-repo-one/runs/7212145369?check_suite_focus=true#step:2:15) (workflow definition [here](https://github.com/test-org-abc123/test-repo-one/actions/runs/2621880767/workflow))

**Additional Notes**
Not sure if this is specific to `prestissimo` plugin, or only in combination with version `7.0` — did not check further.

**Are you willing to submit a PR?**
No — I am not familiar with the action internals.

**Comment by shivammathur** [comment]
Fixed in 6cd4c5330814aeabc8f1216cbfc3f48842d2ed2d

**Comment by shivammathur** [comment]
Fixed in [2.20.1](https://github.com/shivammathur/setup-php/releases/tag/2.20.1)
